wicket-common/src/inventory.rs

```diff
@@ -99,13 +102,48 @@ pub struct Transceiver {
     /// The port in which the transceiver sits.
     pub port: String,
     /// The general status of the transceiver, such as presence and faults.
-    pub status: Option<ExtendedStatus>,
+    #[serde(
+        serialize_with = "serialize_snake_case_result",
+        deserialize_with = "deserialize_snake_case_result"
```

Comment:
Could we squash this down to

```suggestion
        with = "snake_case_result"
```

if we reorganized `serde-helpers` slightly to export a `snake_case_result` module with public `{serialize,deserialize}` functions? (https://serde.rs/field-attrs.html#with)

Reply:
Unfortunately, serde and schemars interpret "with" slightly differently, but I believe that we impl `Serialize`, `Deserialize`, and `JsonSchema` on SnakeCaseResult and do `#[serde(with = "SnakeCaseResult<ExtendedStatus, String>"]

Reply:
Done in 52bd108a4